app/scripts/modules/core/widgets/spelText/spelText.decorator.js

```diff
@@ -20,30 +20,34 @@ let decorateFn = function ($delegate, jsonListBuilder, spelAutocomplete) {
       }
 
       // the textcomplete plugin needs input texts to marked as 'contenteditable'
+      let spelEnabled = !el.attr('class').contains('nospel');
```

Comment:
`.closest()` checks both the element and its ancestors, so this would work if the class were on the `input` or some parent of the input.